treeple/stats/utils.py

```diff
@@ -234,18 +234,19 @@ def _compute_null_distribution_coleman(
 
     # generate the random seeds for the parallel jobs
     ss = np.random.SeedSequence(seed)
-    out = Parallel(n_jobs=n_jobs)(
-        delayed(_parallel_build_null_forests)(
-            y_pred_ind_arr,
-            n_estimators,
-            all_y_pred,
-            y_test,
-            seed,
-            metric,
-            **metric_kwargs,
+    with parallel_config("multiprocessing"):
+        out = Parallel(n_jobs=n_jobs)(
+            delayed(_parallel_build_null_forests)(
+                y_pred_ind_arr,
+                n_estimators,
+                all_y_pred,
+                y_test,
+                seed,
+                metric,
+                **metric_kwargs,
+            )
+            for i, seed in zip(range(n_repeats), ss.spawn(n_repeats))
```

Comment:
Why was this change made?

Reply:
If I remember correctly the default loky would segfault during unit testing the *Oblique trees.